src/context/types.js

```diff
@@ -1,6 +1,7 @@
-export const ADD_TRANSACTION = 'ADD_TRANSACTION';
-export const SET_ASSET_TYPES = 'GET_ASSET_TYPES';
-export const ADD_ASSET_TRANSACTIONS = 'ADD_ASSET_TRANSACTIONS';
+export const ADD_TRANSACTION = "ADD_TRANSACTION";
+export const SET_ASSET_TYPES = "GET_ASSET_TYPES";
+export const ADD_ASSET_TRANSACTIONS = "ADD_ASSET_TRANSACTIONS";
+export const UPDATE_ERROR = "UPDATE_ERROR";
```

Comment:
if you do decide to change `updateError` -> `showError` just remember to update it everywhere

Reply:
I did